File: 1.Basics/2.Patterns/1.go
package main

import "fmt"

func printPattern(n int) {
	for i := 0; i < n; i++ {
		var row string = ""
		for j := 0; j < n; j++ {
			row += "*"
		}
		fmt.Println(row)
	}
}

func main() {
	printPattern(3)
}

File: 1.Basics/2.Patterns/1.js
/*
  Square pattern (N=3)
    * * *
    * * *
    * * *
  
  Rows : outer loop
  Columns : inner loop
*/

/**
 *
 * @param {number} n
 */
function printPattern(n) {
  for (let i = 0; i < n; i++) {
    let row = "";
    for (let j = 0; j < n; j++) {
      row += "*";
    }
    console.log(row);
  }
}

printPattern(2);
